azure-communication-ui/build.gradle

```diff
@@ -26,14 +26,7 @@ buildscript {
         androidx_lifecycle_viewmodel_ktx_version = '2.4.1'
         androidx_navigation_fragment_ktx_version = '2.4.2'
 
-        if (project.rootProject.file('private_drop.properties').canRead()) {
-            Properties privateDropProps = new Properties()
-            privateDropProps.load(project.rootProject.file('private_drop.properties').newDataInputStream())
-            azure_calling_sdk_version = privateDropProps.getProperty('PRIVATE_DROP_VERSION', '""')
-        }
-        else {
-            azure_calling_sdk_version = '2.1.0-beta.1'
-        }
+        azure_calling_sdk_version = '2.2.0'
```

Comment:
We do not need to delete this code
we only need to update version:
azure_calling_sdk_version = '2.2.0'